ios/CodePush/CodePush.m

```diff
@@ -448,7 +435,29 @@ - (void)savePendingUpdate:(NSString *)packageHash
 RCT_EXPORT_METHOD(getConfiguration:(RCTPromiseResolveBlock)resolve
                           rejecter:(RCTPromiseRejectBlock)reject)
 {
-    resolve([[CodePushConfig current] configuration]);
+    NSDictionary *configuration = [[CodePushConfig current] configuration];
+    NSError *error;
+    if (isRunningBinaryVersion) {
+        // isRunningBinaryVersion will not get set to "YES" if running against the packager.
+        NSString *binaryHash = [CodePushUpdateUtils getHashForBinaryContents:[CodePush binaryBundleURL] error:&error];
+        if (error) {
+            NSLog(@"Error obtaining hash for binary contents: %@", error);
+            resolve(configuration);
+            return;
+        }
+        
+        if (binaryHash == nil) {
```

Comment:
Is this accommodating the scenario where you built the app against the simulator and it didn't create the bundle? I would love to add a simple comment to these kinds of statements, since it will be hard to remember what it's purpose is in the future.